github/teams.go

```diff
@@ -355,3 +355,122 @@ func (s *TeamsService) ListUserTeams(ctx context.Context, opt *ListOptions) ([]*
 
 	return teams, resp, nil
 }
+
+type ListTeamProjectsOptions struct {
+	// Affiliation specifies how collaborators should be filtered by their affiliation.
+	// Possible values are:
+	//     outside - All outside collaborators of an organization-owned repository
+	//     direct - All collaborators with permissions to an organization-owned repository,
+	//              regardless of organization membership status
+	//     all - All collaborators the authenticated user can see
+	//
+	// Default value is "all".
+	Affiliation string `url:"affiliation,omitempty"`
+
+	ListOptions
+}
+
+// ListTeamProjects lists the organization projects for a team
+//
+// GitHub API docs: https://developer.github.com/v3/teams/#list-team-projects
+func (s *TeamsService) ListTeamProjects(ctx context.Context, id int64, opt *ListTeamProjectsOptions) ([]*Project, *Response, error) {
+	u := fmt.Sprintf("teams/%v/projects", id)
+	u, err := addOptions(u, opt)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	// TODO: remove custom Accept header when this API fully launches.
+	acceptHeaders := []string{mediaTypeNestedTeamsPreview, mediaTypeProjectsPreview}
+	req.Header.Set("Accept", strings.Join(acceptHeaders, ", "))
+
+	var projects []*Project
+	resp, err := s.client.Do(ctx, req, &projects)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return projects, resp, nil
+}
+
+// ReviewTeamProjects checks whether a team has read, write, or admin
+// permissions for an organization project.
+//
+// Github API docs: https://developer.github.com/v3/teams/#review-a-team-project
```

Comment:
s/Github/GitHub/

Reply:
This one on line 386 still needs changing.

Reply:
Sorry, @palash25 - could you please correct the "GitHub" spelling on line 386?
We like to be consistent in our repo and want to honor the trademark that has been established.

Reply:
sorry @gmlewis not sure how I missed this but its fixed now :sweat_smile: